<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralCrackFrameworkGUIClass</class>
 <widget class="QMainWindow" name="GeneralCrackFrameworkGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>605</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>437</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>605</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>General Crack Framework</string>
  </property>
  <property name="windowIcon">
   <iconset resource="generalcrackframeworkgui.qrc">
    <normaloff>:/GeneralCrackFrameworkGUI/Resources/ico.bmp</normaloff>:/GeneralCrackFrameworkGUI/Resources/ico.bmp</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>446</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>446</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/GeneralCrackFrameworkGUI/Resources/image.jpg);</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_Script">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Load Crack Script</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Path">
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_TargetFile">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Target File</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_process">
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Process</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Target">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>88</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Parameters:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Param"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Crack">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>64</height>
       </size>
      </property>
      <property name="text">
       <string>Crack!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionAbout_General_Crack_Framework">
   <property name="text">
    <string>About General Crack Framework</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="generalcrackframeworkgui.qrc"/>
 </resources>
 <connections/>
</ui>
